gui/wxpython/datacatalog/tree.py

```diff
@@ -293,9 +244,22 @@ def match(self, **kwargs):
         if not kwargs:
             return False
 
-        for key in kwargs:
-            if not (key in self.data and self.data[key] == kwargs[key]):
-                return False
+        if exact:
+            for key in kwargs:
+                if not (key in self.data and self.data[key] == kwargs[key]):
+                    return False
```

Comment:
Probably doesn't matter in speed for small dict, but `key, value in kwargs.items()` is probably better in general to express this anyway and results in better code I think: `self.data[key] == value`.